frame/benchmarking/src/lib.rs

```diff
@@ -418,162 +419,226 @@ macro_rules! benchmarks_iter {
 #[doc(hidden)]
 macro_rules! benchmark_backend {
 	// parsing arms
-	( { $( $instance:ident )? } $name:ident {
-		$( $where_clause:tt )*
-	} {
-		$( $common:tt )*
-	} {
-		$( PRE { $( $pre_parsed:tt )* } )*
-	} { $eval:block } {
+	(
```

Comment:
These macro changes are just superficial. I was trying to edit them at some point, and they were impossible to read to me. So I formatted them how I would expect to seem them. No logic actually changes in the macro parsing